frontend-ui/src/utils/offliner.ts

```diff
@@ -272,7 +272,7 @@ export function artifactsUrl(task: Task) {
   return uploadUrl(task.upload.artifacts.upload_uri, task.container.artifacts)
 }
 
-function uploadUrl(uri: string, filename: string) {
```

Comment:
For consistency, prefer to not export this function and move getCheckUrl function to this file.